### ISSUE TITLE
Measure the tactical floor with forced-material-win probes

### ISSUE THREAD

**Issue by mmmfrieddough** [mmmfrieddough/anthro-chess#94]
## Goal

Measure whether the model finds forced material wins, and whether that ability
survives in positions unlike those humans reach. Legality and move-prediction
metrics both pass on positions where it misses a free piece, so nothing
currently planned would catch this failure.

## Motivating Evidence

In a played game the model was offered the same material win twice in
consecutive positions. When the capture was an ordinary developing recapture it
ranked the three winning captures 1st, 2nd, and 3rd. One ply later, with a
queen hanging on a back-rank square in a position humans never reach, the
winning capture ranked 6th at 3.7% and it castled instead.

Raw-logit legality at that second position was unremarkable, so a legality
slice would have reported nothing. The distinguishing property was not
difficulty or legality but how closely the position resembled training data.

## Scope

An offline probe over derived positions from the frozen pool, reported as its
own metric family. Not a tactics trainer, not engine-strength measurement, and
not a substitute for rollouts.

## Acceptance Criteria

- Probe positions are derived from the frozen pool through the shared view
  layer rather than hand-authored, so the measurement stays on real data and
  the selection is recorded in the artifact.
- Position selection uses exact chess logic to identify forced material gain,
  with the criterion stated explicitly and applied deterministically.
- Results report whether the winning move is the argmax, its rank, and the
  probability mass on winning moves, so a near-miss is distinguishable from an
  absence.
- Positions are sliced by a stated familiarity proxy so the in-distribution and
  unusual cases are reported separately rather than averaged.
- The familiarity proxy is defined from data already available offline and its
  limitations are documented; it should not require a second model.
- Results carry checkpoint, pool, view, action, encoding, and benchmark
  provenance like other benchmark artifacts.
- Fixture tests cover position selection, rank and mass aggregation, and
  deterministic output.

## Notes

Material value alone is a crude criterion and will admit positions where the
capture is unsound for tactical reasons. Whether that noise is tolerable, or
whether the criterion needs to require the gain survive a shallow forced
sequence, should be settled during implementation against real pool positions
rather than assumed here.

The familiarity proxy is the hard design question. Raw-logit legality
correlates with novelty and is already computed, but the motivating example is
exactly a case where it did not. Position frequency in the training corpus,
distance from the nearest opening-book line, and ply-depth from a known line
are all candidates.

**Comment by mmmfrieddough** [comment]
## Source games for the motivating example

Preserving the two hand-played games the motivating evidence came from, so the
positions remain available when this is implemented. Both were played through
the UCI interface against the baseline checkpoint at target rating 2500 and
temperature 0.5, engine as White.

Game 1, an ordinary game:

```text
1.e4 e5 2.Nf3 Nc6 3.d4 d6 4.Bb5 Bd7 5.Bxc6 Bxc6 6.dxe5 Qd7 7.O-O O-O-O 8.c3 Bxe4
9.exd6 Qxd6 10.Nbd2 Nf6 11.Re1 Qc6 12.Nc4 Rxd1 13.Rxd1 Bxf3 14.gxf3 Qxf3
15.Be3 Ng4 16.Bd4 Qxf2+ 17.Kh1 Qxh2# 0-1
```

Game 2, with deliberately poor opponent moves:

```text
1.e4 c6 2.Nf3 Qa5 3.c3 Qxc3 4.Bc4 Qxc1 5.O-O Qxd1 6.Rxd1 Kd8 7.Bb3 Kc7 8.Bc4 Kb6
9.Bb3 Kb5 10.d4 a5 11.Ne5 a4 12.Bxa4+ Kxa4 13.Nc3+ Kb4 14.Nd5+ Kb5 15.Nc7+ Kb6
16.Nxa8+ Ka7 17.Nb6 Kxb6 18.f4 c5 19.f5 c4 20.Rab1 c3 21.d5 c2 22.f6 cxb1=Q
23.Kf2 Qxd1 24.Ke3 Qxd5 25.exd5 g5 26.g3 g4 27.Kd4 h5 28.Ke4 h4 29.d6 hxg3
30.hxg3 Rh3 31.Nxd7+ Bxd7 32.Ke5 Rxg3 33.Kf4 Rg1 34.Ke4 g3 35.Ke5 Re1+ 36.Kd5 g2
37.Kc4 g1=Q 38.Kb3 Qe3+ 39.Kc4 Qe4+ 40.Kb3 Re3# 0-1
```

The two positions that motivate this issue are consecutive decisions in game 2.

Ordinary-looking capture, `rnb1kbnr/pp1ppppp/2p5/8/4P3/2q2N2/PP1P1PPP/RNBQKB1R w KQkq - 0 4`:

```text
1. bxc3  28.38%   <- wins the queen
2. dxc3  13.19%   <- wins the queen
3. Nxc3  10.87%   <- wins the queen
```

One ply later, `rnb1kbnr/pp1ppppp/2p5/8/2B1P3/5N2/PP1P1PPP/RNqQK2R w KQkq - 0 5`:

```text
1. O-O    22.42%
2. Ke2    17.81%
3. Nc3    15.25%
5. Bxf7+   4.07%
6. Qxc1    3.70%   <- wins the queen
```

Raw-logit mask penalty at the second position was 0.4823, unremarkable against
a held-out middlegame average near 0.91. Every legality metric passes there.

These are useful as fixtures but should not become the probe set. The point of
this issue is deriving probe positions from the pool rather than hand-picking
them; these two exist to check that an implementation reports something
sensible on a case already understood.

**Comment by mmmfrieddough** [comment]
Folded into #96.

These were the same measurement approached from opposite directions. This issue tried to *detect* novelty on natural pool positions with a familiarity proxy; #96 *generates* it by perturbation at a known dose. Detection turned out not to work, for a more basic reason than the weakness of any individual proxy: the pool is human games, so it is in distribution nearly by construction, and the position that motivated this issue came from the model's own play against a human rather than from any pool game. No slice over the pool can reach it.

Everything from this issue is carried into #96 rather than dropped:

- the motivating evidence, in full;
- the material-gain probe, reported at every dose alongside the adjudicated predicates from #99;
- argmax, rank, and probability-mass reporting so a near-miss stays distinguishable from an absence;
- the criterion-soundness question, which stops being blocking because results are reported only relative to a reference — the human rate at dose zero, the model's own unperturbed rate above it — and a reference absorbs criterion noise as long as the criterion is deterministic and identically applied;
- the in-distribution versus unusual split, now the dose axis rather than a proxy slice.

The only thing not carried over is the familiarity proxy itself and its candidate list, which the dose axis replaces. The reasoning for dropping it is recorded in `docs/evaluation.md` under Novelty so it is not attempted again.
